### ISSUE TITLE
Список заказов очень долго грузится

### ISSUE THREAD

**Issue by sashabeep** [Serg28/tsvshop#7]
При нескольких тысячах заказов, что вполне реально набрать в какой-нибудь службе доставки за месяц, плагин грузится титанически долго, потому что загружает все заказы разом. Нужно что-то с этим сделать

**Comment by Serg28** [comment]
Да, есть такое. Пока ЕВО активно меняется от версии к версии, глобально менять модуль не будем, но  список заказов поправим в ближайшее время.
